src/billboard.py

```diff
@@ -0,0 +1,7 @@
+"""Implementation of Name on Billboard Kata on CodeWars"""
+from numpy import multiply
+
+
+def billboard(name, price=30):
+    """Return cost of printing name on billboard."""
+    return multiply(len(name), price)
```

Comment:
This is what you needed numpy's "multiply" for? You couldn't just use the `*` operator?